### PR TITLE
Update to Lesson Design episode

### LINKED ISSUE #364: Trainee feedback: Lesson Design & Target Audience sections
### How could the content be improved?

An excerpt from a minute card feedback response:

> Some of the background knowledge felt a little disconnected from the content of the lesson. Maybe reflow how much of the Nicholls phases can be recast as organizational structure for the schedule of the lesson, and redirect some of the content into examples (e.g. expertise levels -> ideating the target audience)

I interpret this as two suggestions for improvement:

1. After introducing the iterative design process, provide an overview of the curriculum for this training and highlight how the training is organised to follow one cycle through that process.
2. (Moved to #435) ~~Provide more examples of exactly what we will be doing at each stage of the process, and more examples generally e.g. by discussing how considerations of expertise level might be captured in your description of target audience.~~

### PR DESCRIPTION
**Pull request by tobyhodges**
Attempt to resolve #364, by re-framing the iterative design process and drawing more direct connections between the individual stages and the structure of the training.

### CODE REVIEW

File: episodes/lesson-design.md [hunk 1]
```diff
@@ -22,79 +22,58 @@ After completing this episode, participants should be able to...
...
 }
 
-Note the cyclical nature of this process:
-you will complete one iteration through this cycle during this training
-(though probably for a limited part of your lesson, rather than the whole thing).
-Note also that teaching the content is an essential intermediate step in the process:
-the importance of feedback gathered while teaching the lesson
-will be a common refrain throughout this training.
+The Carpentries community specialises in relatively short-format workshops, typically lasting less than a week and including no opportunities for any assessment after the teaching has finished (a _summative_ assessment).
+To account for this, our lessons (and this training) place an emphasis on _formative_ assessment: assessment of learner progress that takes place _while the teaching is still going on_, to give instructors opportunities to evaluate the teaching and lesson content before the end of the workshop.
```
Comment (sstevens2) [inline episodes/lesson-design.md:45]: This seems out of place with the flow. Do we need it here? If so, maybe need to tie to the material above and below, which seem to be related already

File: episodes/lesson-design.md [hunk 2]
```diff
@@ -22,79 +22,58 @@ After completing this episode, participants should be able to...
...
+2. Design assessments to determine progress towards desired outcomes. (Part 1)
+    * You will learn about different types of assessment and how they can give you information about your learners' progress towards the defined objectives.
+    * You will design and implement exercises that are appropriate to your target audience and the skills you want to teach them.
+3. Write content to lead learners from one of these assessments to the next. (Part 1)
+    * You will choose examples and a narrative that can help learners gain insight into the topic of your lesson.
+    * You will begin to write content that is accessible, relevant, and appropriate for your target audience.
+4. Assess learner progress towards outcomes during teaching. (During the break between parts)
+    * You will deliver part of your new lesson and gather information about how effectively it teaches learners what they need to know.
+5. Evaluate how closely the outcomes meet the objectives. (Part 2)
+    * You will revisit your lesson design and content and make plans to update it, based on your own reflections and the feedback you gathered from learners.
```
Comment (sstevens2) [inline episodes/lesson-design.md:61]: Love this section!

**Review by sstevens2 — APPROVED** [review]
One small comment on the assessment paragraphs. But looks really good!  Love the simplification of the drawing and the explicit connection between the stages and what we do in this training!

File: episodes/lesson-design.md [hunk 3]
```diff
@@ -22,79 +22,58 @@ After completing this episode, participants should be able to...
...
-Note also that teaching the content is an essential intermediate step in the process:
-the importance of feedback gathered while teaching the lesson
-will be a common refrain throughout this training.
+The Carpentries community specialises in relatively short-format workshops, typically lasting less than a week and including no opportunities for any assessment after the teaching has finished (a _summative_ assessment).
+To account for this, our lessons (and this training) place an emphasis on _formative_ assessment: assessment of learner progress that takes place _while the teaching is still going on_, to give instructors opportunities to evaluate the teaching and lesson content before the end of the workshop.
+
+### The process throughout this training
+Note the cyclical nature of the process described above. You will complete one iteration through this cycle across the two parts of this training (though probably for a limited part of your lesson, rather than the whole thing):
+
+1. Define desired learning outcomes. (Training part 1)
```
Comment (anenadic) [inline episodes/lesson-design.md:50]:
```suggestion
1. Define desired learning outcomes. (Part 1)
```

Comment (anenadic) [reply episodes/lesson-design.md:50]: For consistency

**Comment by tobyhodges** [comment]
[828b3f9](https://github.com/carpentries/lesson-development-training/pull/479/commits/828b3f997862ba8c6725b42c246e276058d1d309) made to address @sstevens2's comment in review.

**Pull request merged**